controllers/shared/RTLConf.js

```diff
@@ -335,3 +304,22 @@ exports.updateServiceSettings = (req, res, next) => {
     });
   }
 };
+
+var maskPasswords = function(obj) {
+  var keys = Object.keys(obj);
+  var length = keys.length;
+  if (length !== 0) {
+    for (var i = 0; i < length; i++) {
+      if (typeof obj[keys[i]] === 'object') {
+        keys[keys[i]] = maskPasswords(obj[keys[i]]);
+      }
+      if (typeof keys[i] === 'string'
+        && (keys[i].toLowerCase().includes('password') || keys[i].toLowerCase().includes('multipass')
+          || keys[i].toLowerCase().includes('rpcpass') || keys[i].toLowerCase().includes('rpcpassword'))
+      ) {
+        obj[keys[i]] = obj[keys[i]].replace(/./g, '*');
```

Comment:
Leaking password length is also not great but I guess whole issue was exaggerated as anyone with access to RTL could just steal sats directly, right?

Reply:
@Kixunil Thanks for the review. 

As you already stated, if someone is able to access upto this layer then the user has bigger issues to worry about than the length of the password. :)

Additionally, this `length` variable is NOT reading the length of the password but only the length of the keys. ie. the number of sub-keys available under that header/key. 

For example, the length will be 3 for below header (`'Bitcoin`) as it has three sub-keys.
[Bitcoin]
bitcoin.active=1
bitcoin.testnet=1
bitcoin.node=bitcoind

Reply:
I meant leaking length by replacing each character with exactly one asterisk as opposed to replacing whole password with fixed-length string. (One can determine password length by counting the number of asterisks.)

Reply:
@Kixunil Thanks for the suggestion. Created a new issue #689 to track it.